x-pack/plugin/eql/qa/common/src/main/resources/mapping-default.json

```diff
@@ -72,8 +74,8 @@
         "file_name" : {
             "type" : "text",
             "fields" : {
-                "keyword" : {
-                    "type" : "keyword",
+                "[keyword_like_type]" : {
+                    "type" : "[keyword_like_type]",
                     "ignore_above" : 256
```

Comment:
Does this setting make sense for wildcard?

Reply:
Yes, it's [the only supported setting of a `wildcard` field type](https://www.elastic.co/guide/en/elasticsearch/reference/master/keyword.html#wildcard-field-type).